index.html

```diff
@@ -160,10 +160,10 @@ <h1>Frequently asked questions</h1>
 				<div class="col-md-5">
 					<div class="faq-question">
 								<h2 class="text-colored">What's so special about HackHPI?</h2>
-								<p>There are very few health tech hackathons striving to build useful applications with raw data, and we want to change that! For you it will be a chance to get in contact with our awesome partners and experiment with their data and gadgets.</p>
+								<p>There are very few hackathons about environment striving to build useful applications with raw data, and we want to change that! For you it will be a chance to get in contact with our awesome partners and experiment with their data and gadgets.</p>
 							</div><div class="faq-question">
 								<h2 class="text-colored">Can I be part of it?</h2>
-								<p>Sure! As long as you are interested in&nbsp;health tech and 18 years or older we will be happy to welcome you! Any skill-level is welcome, hackathons should be a way to experience team-work and work on your own abilities.</p>
+								<p>Sure! As long as you are interested in&nbsp;our environment and 18 years or older we will be happy to welcome you! Any skill-level is welcome, hackathons should be a way to experience team-work and work on your own abilities.</p>
```

Comment:
legacy typo: 

> and **are** 18 years or (...)

Reply:
Not an error, just a thing of personal writing style...